fluentui_transients/src/main/java/com/microsoft/fluentui/tooltip/Tooltip.kt

```diff
@@ -173,12 +177,15 @@ class Tooltip {
         tooltipBackGround.background = ContextCompat.getDrawable(context, drawable)
     }
 
-    fun dismiss() {
+    private fun dismissSideEffects() {
         tooltipView.announceForAccessibility(context.getString(R.string.tooltip_accessibility_dismiss_announcement))
-        popupWindow.dismiss()
         onDismissListener?.onDismiss()
     }
 
+    fun dismiss() {
+        popupWindow.dismiss()
```

Comment:
we will have to call listener from here too

Reply:
Yes, I have attached the listener there, Line 160. So from here we dismiss the popupWindow and then it calls the dismissSideEffects.